README.md

```diff
@@ -16,13 +16,13 @@ Please install the package with Julia's package manager:
 ] add CDSAPI
 ```
 
-## Usage
+## Basic usage
 
-Make sure your `~/.cdsapirc` file exists. Instructions on how to create the file for your user account can be found
+Make sure your `~/.cdsapirc` file exists or the env vars `CDSAPI_URL` and `CDSAPI_KEY` are set.
```

Comment:
This paragraph could be expanded to discuss the precedence order of the different credential methods. I believe we could copy/paste the docstring of `CDSAPI.credentials` here with minor adjustments to improve the flow.